assets/script/script.js

```diff
@@ -8,26 +11,91 @@ fetch("https://api.football-data.org/v2/competitions/PL/teams", {
     return response.json();
   })
   .then(function (data) {
-    footballData = data;
+    teamData = data;
   })
   .then(renderCrests)
-  .then(renderTeamInfo);
+  .then(fillTagsArray);
 
+// get data for league standings
+fetch("https://api.football-data.org/v2/competitions/PL/standings", {
+  headers: { "X-Auth-Token": "b7bac95afb44489a83d8eb77fc894151" },
+})
+  .then(function (response) {
+    return response.json();
+  })
+  .then(function (data) {
+    standings = data;
+  });
 
 // adds all crests to the welcome page
 function renderCrests() {
-  for (i = 0; i < footballData.teams.length; i++) {
+  for (let i = 0; i < teamData.teams.length; i++) {
     $("#crestContainer").append(
```

Comment:
Might be worth a `const team = teamData.teams[i]` here: it would make for DRYer code.